xchange-core/src/main/java/org/knowm/xchange/service/polling/trade/PollingTradeService.java

```diff
@@ -32,19 +33,34 @@
 
   /**
    * Gets the open orders
-   * 
+   *
    * @return the open orders, null if some sort of error occurred. Implementers should log the error.
    * @throws ExchangeException - Indication that the exchange reported some kind of error with the request or response
    * @throws NotAvailableFromExchangeException - Indication that the exchange does not support the requested function or data
    * @throws NotYetImplementedForExchangeException - Indication that the exchange supports the requested function or data, but it has not yet been
    *         implemented
    * @throws IOException - Indication that a networking error occurred while fetching JSON data
    */
-  public OpenOrders getOpenOrders() throws ExchangeException, NotAvailableFromExchangeException, NotYetImplementedForExchangeException, IOException;
+  OpenOrders getOpenOrders() throws ExchangeException, NotAvailableFromExchangeException, NotYetImplementedForExchangeException, IOException;
```

Comment:
Yeah but that will break the API for users. What about marking getOpenOrders {@Deprecated} and mark for removal in the future?